corehq/apps/app_manager/models.py

```diff
@@ -2067,6 +2067,9 @@ class Detail(IndexedSchema, CaseListLookupMixin):
     # Custom variables to add into the <variables /> node
     custom_variables = StringProperty(exclude_if_none=True)
 
+    # Allow selection of mutiple cases
```

Comment:
nit: update comment to indicate where this applies

```suggestion
    # Allow selection of mutiple cases. Only applies to 'short' details
```